packages/amplify-util-uibuilder/src/__tests__/getAmplifyDataSchema.test.ts

```diff
@@ -2,134 +2,84 @@ import aws from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependenc
 import { AmplifyCategories, AmplifySupportedService, FeatureFlags } from 'amplify-cli-core'; // eslint-disable-line import/no-extraneous-dependencies
 import { printer } from '@aws-amplify/amplify-prompts'; // eslint-disable-line import/no-extraneous-dependencies
 import { mocked } from 'ts-jest/utils';
-import { AmplifyStudioClient } from '../clients';
 import { getAmplifyDataSchema } from '../commands/utils';
 
-const awsMock = aws as any;
-const printerMock = printer as any;
+jest.mock('amplify-prompts');
```

Comment:
please address codeql warnings in this file.

Reply:
Thanks, addressed